docs/military/townsville.md

```diff
@@ -24,16 +24,16 @@ TL TCU owns the Class C and G airspace within 36 DME TL from `SFC` to `F180`
 </figure>
 
 ### ADC
-TL ADC owns the Class C arispace in the TL CTR (Extends to 7NM of the thresholds of 01 and 19.) from `SFC` to `A015`. This airsapce is designed to facilitate Helicopter Traffic and Ciruit Traffic.
+TL ADC owns the Class C airspace in the TL CTR (Extends to 7NM from the thresholds of Runway 01/19) from `SFC` to `A015`. This airsapce is designed to facilitate Helicopter Traffic and Ciruit Traffic.
 
 ## Aerodrome
 ### Runway Operations
-Runways 01 and 19 are the primary runways at Townsville. 
-Runway 07 is often used in conjunction with 01 for VFR arrivals and Helicopter Traffic. 
+Runway 01/19 are the primary runways at Townsville. 
+Runway 07/25 is often used in conjunction with 01 for VFR arrivals and Helicopter Traffic. 
```

Comment:
I guess if we put 25 there, we'd need to reference 19 as well (you wouldn't use 25 and 01 together) which becomes redundant, so maybe just 'Runway 07/25 is often used for VFR arrivals and helicopter traffic.'